src/features/search/pages/__tests__/useShowResultsForCategory.test.ts

```diff
@@ -4,7 +4,7 @@ import { SearchGroupNameEnum } from 'api/gen'
 import { LocationType } from 'features/search/enums'
 import { initialSearchState } from 'features/search/pages/reducer'
 
-import { navigate } from '../../../../../__mocks__/@react-navigation/native'
+import { push } from '../../../../../__mocks__/@react-navigation/native'
```

Comment:
est-ce que ceci fonctionnerait ?

```suggestion
import { push } from '__mocks__/@react-navigation/native'
```